AISKU/scripts/listCdnVersions.ps1

```diff
@@ -429,28 +432,59 @@ Function ListVersions(
 ) {
 
     $sortedKeys = $files.Keys | Sort-Object
+    $orderedKeys = New-Object 'system.collections.generic.list[string]'
     foreach ($key in $sortedKeys) {
+        $verParts = $key.split(".");
```

Comment:
Additionally ordering the versions so that all of the major/minor versions (v2; v2.x) versions are listed before the source full versions, so you get 
v2
v2.4
v2.5
v2.4.0
v2.4.1
v2.5.0
v2.5.1
etc